frontend/pages/game_page.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tetris</title>
+    <link rel="stylesheet" type="text/css" href="game_page.css">
+
+</head>
+<body>
+    <h1>
+        <span class="red">T</span>
+        <span class="green">E</span>
+        <span class="blue">T</span>
+        <span class="magenta">R</span>
+        <span class="purple">I</span>
+        <span class="orange">S</span>
+    </h1>
+
+    <h2>
+        Your Score is: 
+    </h2>
+    
```

Comment:
Could we add a div element here containing a (blank) HTML canvas for the game board?